packages/compass-saved-aggregations-queries/src/stores/copy-to-clipboard.ts

```diff
@@ -75,7 +47,7 @@ export function copyToClipboard(
 
     const copyStr =
       item.type === 'aggregation'
-        ? formatPipeline(item.aggregation.pipeline)
+        ? item.aggregation.pipelineText
```

Comment:
Drive-by: we forgot to adjust for the changes in the stored pipeline format, no need to build the string here anymore, we can just return `pipelineText` directly